### PR TITLE
Iss82

### PR DESCRIPTION
**Pull request by kobyingram11**
k

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
+# For the full list of supported browsers by the Angular framework, please see:
+# https://angular.io/guide/browser-support
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
```
Comment (mjohnson7cd) [inline .browserslistrc:9]: dude check your whatever

**Review by KatrinaYates — APPROVED** [review]
Looks great! I am so proud!

**Review by mjohnson7cd — APPROVED** [review]

**Review by DrFlamngo — APPROVED** [review]
very awesome

**Review by zanemooney — APPROVED** [review]
this is pretty cool beanz thank you for showing me this

**Review by kobyingram11 — COMMENTED** [review]
Double check your code

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
```
Comment (esthergiles) [inline .editorconfig:8]: kewl

**Review by zachcutshall — APPROVED** [review]
Looks good

**Review by esthergiles — APPROVED** [review]
very neat

**Review by nikkisteww — APPROVED** [review]

**Review by duncanate — COMMENTED** [review]
Great job! Keep it up!

**Review by vanpham2000 — APPROVED** [review]

**Review by urielkodia — APPROVED** [review]

**Review by brandoncoplen — COMMENTED** [review]
nice job Cumbie

**Comment by vanpham2000** [comment]
yeah

**Review by maceyraejones — APPROVED** [review]
hey

**Comment by urielkodia** [comment]
Nice

**Review by Meluvy — COMMENTED** [review]
mel wuz here

**Pull request merged**